app/CHANGELOG.md

```diff
@@ -1,196 +1,9 @@
-## What's Changed in app@0.20.1
-* fix: run deeplink on continue user activity by @fortmarek in [#7897](https://github.com/tuist/tuist/pull/7897)
+# Changelog
 
-**Full Changelog**: https://github.com/tuist/tuist/compare/app@0.20.0...app@0.20.1
+All notable changes to this project will be documented in this file.
+## What's Changed
```

Comment:
I adjusted the configuration of git cliff to filter commits with the conventional commit scope "app" which maps to another CHANGELOG.md. Since the current content of this one is already persisted in GitHub releases, my suggestion is that we reset the content of the file to the next version using the new convention.